app/models/manageiq/providers/openstack/cloud_manager/cloud_tenant.rb

```diff
@@ -4,4 +4,17 @@ class ManageIQ::Providers::Openstack::CloudManager::CloudTenant < ::CloudTenant
                           :join_table              => "cloud_tenants_vms",
                           :association_foreign_key => "vm_id",
                           :class_name              => "ManageIQ::Providers::Openstack::CloudManager::Template"
+
+  def self.post_refresh_ems(ems_id, _)
+    ems = ExtManagementSystem.find(ems_id)
+
+    MiqQueue.put(
```

Comment:
This (putting it on the refresh worker queue of the provider) looks correct considering that the work we're queueing here is related to the provider and it's a post refresh operation. However, I'm not sure if it would be ok or even better to just put this on the generic queue since it will not have to communicate with the provider directly to do the work. @Fryguy any thoughts or opinions on this?